db/schema.rb

```diff
@@ -19,6 +19,8 @@
     t.string   "icon_url"
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
+    t.string   "rating"
```

Comment:
No easy way that I know of, no... What you could do is branch off from the commit behind the one you want to change, and cherry pick the commits one by one, amending them as needed. In our repo, I wouldn't really bother :P